codecov.yml

```diff
@@ -0,0 +1,15 @@
+comment:
+  layout: header, changes, diff, sunburst
+coverage:
+  status:
+    patch: false
+    # project will give us the diff in the total code coverage between a commit
+    # and its parent
+    project:
+      default:
+        target: auto
+        threshold: "0.05%"
+    changes: false
+ignore:
+  - "**/internal/test"
+  - "vendor/*"
```

Comment:
Are there any mocks you'd want to ignore? Also integration test modules (though I'm assuming internal/test refers to your system tests)